tutorials/gl/glbox.C

```diff
@@ -2,10 +2,12 @@
 /// \ingroup tutorial_gl
 /// Display a 3D histogram using GL (box option).
 ///
-/// \macro_image(nobatch)
```

Comment:
@couet, could you please confirm that this change is good?

Reply:
no it is not, \author must be the last directive of the tutorial's headers.

Reply:
see the last line of this page: https://root.cern/for_developers/docs/guidelines/#tutorials-documentation